flytekit/core/map_task.py

```diff
@@ -1,72 +1,168 @@
-from typing import Any, Optional
+import os
+from typing import Any, Dict, List, Optional, Type
 
+from flytekit.common.tasks.raw_container import _get_container_definition
 from flytekit.core.base_task import PythonTask, TaskMetadata
+from flytekit.core.context_manager import ExecutionState, FlyteContext, SerializationSettings
 from flytekit.core.interface import transform_interface_to_list_interface
+from flytekit.core.python_function_task import PythonFunctionTask, get_registerable_container_image
+from flytekit.models.interface import Variable
+from flytekit.models.task import Container
 
 
 class MapPythonTask(PythonTask):
     """
```

Comment:
lets have dome docs on the usage and explaining what concurrency, min_success_ratio etc means.

Reply:
done in the exported `maptask` below